docker-compose.yaml

```diff
@@ -54,6 +54,7 @@ services:
       - host.docker.internal:host-gateway
   postgres:
     image: postgres:15
+    command: -c 'max_connections=200'
```

Comment:
Adding this has fixed the e2e test that was failing always:
https://github.com/trento-project/web/actions/runs/12478685578/job/34826632861

It creates a `max_connections` hit error, the host takes a while to roll up, and the events sent to this host are excluded as it is in `rolling_up` state.